docs/js-core-3/week-2/lesson.md

```diff
@@ -398,6 +399,41 @@ fetch(_____)
 
 :::
 
+## PD (Agile: Planning and preparation)
+
+**Session objective:** In this session we will talk about refining user story backlog, planning the sprint and estimating effort.
+
+:::note Warm-up (10 minutes)
+
+Write a mission statement for the TV show project:
+
+- What's the purpose of the project?
+- What would be the mission of the sprints?
+
+:::
+
+[Agile: Planning and preparation](https://docs.google.com/presentation/d/1TJNjPOha2P5cWW6IO9jAc9GTvsosqVGkkG5pAbbvfg4/edit#slide=id.g1b740a9f8c6_0_336)
+
+:::note Exercise (30 minutes)
+
+**Exercise objective:** To practice estimating effort for user stories
+
+In small groups. For this sizing example, you have 5 stories. The first one is already t-shirt sized as a Medium. To size story #2, the team must compare it against story #1 - just as you’d compare t-shirts to find their size - Is it bigger or smaller?
```

Comment:
Work in small groups - should we say at least 4 people?

Reply:
> Work in small groups - should we say at least 4 people?

We used to name a figure, but in practice, the cohorts have different needs and arrangements. I prefer to leave it a bit flexible instead of imposing a strict number. I can change it if you want me to :)

Reply:
For me, the strange thing is how the phrase is built: "In small groups." . So maybe just rewrite it, so we don't start with a phrase that doesn't reflect a good English sentence.

Reply:
Done.